opm/upscaling/CornerpointChopper.hpp

```diff
@@ -379,7 +379,7 @@ namespace Opm
             if (data.empty())
                 return;
 
-            Opm::DeckKeyword dataKw(&keyword);
+            Opm::DeckKeyword dataKw(keyword);
```

Comment:
You probably need to make the same change on [line 359](https://github.com/OPM/opm-upscaling/blob/1bc89ef7aae6265bd149be8a244c183c44a853ab/opm/upscaling/CornerpointChopper.hpp#L359) (in the `addDoubleKeyword()` member function).

Reply:
Yes - thank you - done. My local opm-upscaling did not build at all - some lapack issue????